src/components/pages/ProfilePage.tsx

```diff
@@ -22,8 +22,8 @@ export const ProfilePage = async () => {
       const { userId } = getCurrentUserIdResult.unwrap();
       const getUserByIdResult = await getUserById(userId);
       if (getUserByIdResult.ok) {
-        const { name } = getUserByIdResult.unwrap();
-        return name;
+        const { username } = getUserByIdResult.unwrap();
+        return username;
```

Comment:
@kanta1207 cc @ShoeheyOt 

Sorry for commenting after merging.

I think the key name defined in the backend is incorrect.

Our [Web API Document](https://nishiki-tech.github.io/nishiki-documents/web-api/index.html#tag/user/paths/~1users~1%7BuserId%7D/get) says the key is `name`, not `username`.

The term `username` sounds like a unique identifier, for example, "kanta1207" on your GitHub account. What we want here is the "name of the user." So, naming it `username` is a bit misleading.

Reply:
@hitohata @taniguchiiqqq 
Could you please confirm this when you have a moment?

Reply:
@nick-y-ito 
I've created a new issue. Sorry for your inconvenience.
https://github.com/nishiki-tech/nishiki-backend/issues/315

Reply:
@nick-y-ito Thank you for bringing it up!

I will quickly change it to `name` from `username` once it's resolved in backend, 
thank you @hitohata for the quick response!

Reply:
@hitohata @kanta1207
Thank you for your prompt actions!